.github/workflows/main.yml

```diff
@@ -91,10 +91,10 @@ jobs:
     - name: Install Python dependencies
       run: |
         python3 -m pip install ruamel.yaml scons==3.1.2 numpy cython pandas pytest \
-        pytest-github-actions-annotate-failures pint graphviz
+        pytest-github-actions-annotate-failures pytest-xdist pint graphviz
```

Comment:
At some point, between these dependencies and the docs, it'll probably make sense to just put a requirements.txt file in the repo